test-app/tests/unit/setup-rendering-context-test.js

```diff
@@ -287,15 +286,21 @@ module('setupRenderingContext', function (hooks) {
     });
 
     test('can invoke helper', async function (assert) {
-      await render(hbs`{{jax "max"}}`);
+      this.setProperties({
+        jax: helper(([name]) => `${name}-jax`),
+      });
+      await render(hbs`{{this.jax "max"}}`);
```

Comment:
these helpers were (rightfully so!) trying to look up the helper defined in app/helpers/jax.

However, that doesn't exist, and since we don't actually support custom resolvers for normal apps, this is fine. These tests are testing that `render` can invoke helpers, and helpers can be defined on `this`